source/variables/custom-selectors.css

```diff
@@ -1 +1,8 @@
-@custom-selector :--enter :hover, :active;
\ No newline at end of file
+/* 
+	Allows you to create your own selectors
+	http://cssnext.io/features/#custom-selectors
+	:--enter {
+	  ... styles ...
+	}	
+*/
+@custom-selector :--enter :hover, :focus, :active;
```

Comment:
I was trying to state that :focus is use for accessibility where :hover & :active styles are for design.

As a low vision user tabbing through the page, I will expect to see focus glow-rings around an element that can be focused.  These are helpful for me to understand where on the page I am focuses.  

Styling hover and active state together makes sense to keep the "hover" state of an element consistent on touch and mouse based devices.  What does :focus mean on a link on touch? On mobile, focus styles are only going to be consumed in text fields, or if you have voiceover turned on.

With all that in mind, we should provide both.  There are times where we want hover and focus to be the same.

```
@custom-selector :--enter :hover, :active;
@custom-selector :--haf :hover, :active, :focus;
```